k8s/openshift/byo/baremetal/README.md

```diff
@@ -211,8 +211,9 @@ Alternatively, the following command may be used:
 oc adm policy add-scc-to-user anyuid -z default --as=system:admin 
 ```
 
-Note: While the above procedures may be sufficient to enable host access to the containers, it may also be needed to disable
-selinux (via ```setenforce 0```) to ensure the same. 
+Note: While the above procedures may be sufficient to enable host access to the containers, it may also be needed to :
+- Disable selinux (via ```setenforce 0```) to ensure the same. 
+- Edit the restricted scc to use ```runAsUser: runAsAny``` (the replica pod runs with root user) 

```

Comment:
One of the users in slack mentioned that the above should be "RunAsAny"

Reply:
yes, will correct the typo !!

Reply:
Done !